src/components/ErrorPane.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    <div class="backdrop" @click="$emit('backdrop-clicked')"></div>
+    <div class="error-pane">
+      <div class="error-title">
+        <h4>Detailed Error Log</h4>
+      </div>
+      <div class="error-text">
+        <pre v-html="colorizedErrorLog"></pre>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import Convert from 'ansi-to-html';
+
+export default {
+  name: 'ErrorPane',
+  props: ['errorLog'],
```

Comment:
if this is a string, you can get some checking by vue by using extended syntax. Something like
errorLog: { type: String } docs will have more details

Reply:
https://vuejs.org/guide/components/props.html#prop-validation